packages/ui/app/src/api-page/types/object/ObjectProperty.tsx

```diff
@@ -1,7 +1,8 @@
-import * as FernRegistryApiRead from "@fern-fern/registry-browser/api/resources/api/resources/v1/resources/read";
+import type * as FernRegistryApiRead from "@fern-fern/registry-browser/api/resources/api/resources/v1/resources/read";
```

Comment:
why this change?

Reply:
When an import is used only for type annotations, I use type-only import since it gets stripped out and is guaranteed to have no effect on runtime.